snapshot-labs/sx-monorepo
# fix: vote modal should close on proposal change

### Summary



Closes: #613 

This PR:

- will always close the vote modal when the proposal has changed
- Will revert back `editMode` to false on the Proposal vote form

### How to test

1. Go to any active proposal, and open the vote modal
2. Change the proposal ID in the url by another one (either also active, or closed)
3. The vote modal should be closed, instead of being left open with stale content
4. If you have already voted on the proposal, the vote form should exit the `editMode` on proposal change
